src/redash_stmo/integrations/iodide/extension.py

```diff
@@ -17,46 +18,51 @@
 logger = logging.getLogger(__name__)
 
 
-class IodidePostResource(BaseResource):
+class IodideNotebookResource(BaseResource):
     @require_permission("view_query")
     def get(self, query_id):
         query = get_object_or_404(
             Query.get_by_id,
             query_id,
         )
-        url = "https://stage.iodide.nonprod.dataops.mozgcp.net/api/v1/notebooks/"
-        template = open(
-            "/extension/src/redash_stmo/integrations/iodide/iodide-template.iomd",
-            "r"
-        )
-        completed_template = template.read().replace(
-            "[QUERY_ID]",
-            query_id
-        ).replace(
-            "[REDASH_API_KEY]",
-            settings.REDASH_OWN_API_KEY_FOR_IODIDE
-        ).replace(
-            "[TITLE]",
-            query.name
-        )
+        with open(os.path.join(os.path.dirname(__file__), 'iodide-notebook.iomd.j2'), "r") as template:
+            source = template.read()
+            context = {
+                "query_id": query_id,
+                "title": query.name,
+                "api_key": settings.IODIDE_DEFAULT_API_KEY,
+            }
+            rendered_template = render_template_string(source, **context)
         headers = {
-            "Authorization": "Token %s" % settings.IODIDE_API_KEY,
+            "Authorization": "Token %s" % settings.IODIDE_AUTH_TOKEN,
         }
         data = {
-            "owner": "jkarahalis@mozilla.com",
+            "owner": "redash",
```

Comment:
Ah, neat, is that something we can use over there?

Reply:
Yes! Although Will suggested that I use the email address of the Redash user. I'll do that instead.